core/src/main/scala/org/http4s/Message.scala

```diff
@@ -273,6 +274,34 @@ final class Request[F[_]](
 
   def queryString: String = uri.query.renderString
 
+  /** cURL representation of the request.
+    *
+    * Supported cURL-Parameters are: -X, -H
+    *
+    */
+  def asCurl(
+      redactHeadersWhen: CaseInsensitiveString => Boolean = Headers.SensitiveHeaders.contains)
+      : String = {
+
+    /**
```

Comment:
I think [this build failure](https://travis-ci.org/http4s/http4s/jobs/642122692#L354) is because it doesn't like scaladoc in an internal method. If the build fails again, a single-asterisk here should clean that up.

Reply:
ah, great! was already confused why doc-build was failing, will do, when next build fails (probably it will)